src/test/java/uk/gov/hmcts/reform/em/orchestrator/endpoint/CcdCloneBundleControllerTest.java

```diff
@@ -30,7 +31,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @RunWith(SpringRunner.class)
-@SpringBootTest
+@SpringBootTest(classes = {Application.class})
```

Comment:
??

Reply:
The main package now has multiple `@SpringBootApplication` classes. The unit tests need to know which one to boot in order to run the tests.

Reply:
makes sense, the CliApplication class :+1: